Review for level 100 -300

Nishka-Kisten opened the pull request.

Nishka-Kisten committed:
Step 12

Nishka-Kisten committed:
level 100 - 300

Nishka-Kisten committed:
updated 100-300

Nishka-Kisten committed:
Update style.css

Nishka-Kisten committed:
update

Nishka-Kisten committed:
Update script.js

chandredewet commented on script.js:
```diff
@@ -1,12 +1,130 @@
+//episode data to be displayed on screen
+function display(){
+  let data = arr.map((ep)=> {
+    prepended_out = ("0" + ep.season ).slice(-2);
```
Investigating slice(-2) here. Maybe an opportunity for me to learn too.  Would ("0" + ep.season) not already be in format you need e.g. 02 or am I missing something. To me this reads that you make 02, then you cut it out and assign it to prepended out.

Nishka-Kisten replied:
I cant remember the exact explanation I found while I was researching to do that.. but basically, ("0" + ep.season), concatenates the 0 I need and the slice slices the last two digits so like.. if its a number greater or equal to 10.. those last two digits will show but less than will add the concatenated 0.. Does that make sense?

chandredewet commented on script.js:
```diff
@@ -1,12 +1,130 @@
+             <img src="${ep.image}" title ="ep"></img>
+              ${ep.summary} </div> ` ;
+                        
+         }).join('');
```
Great use of .joining, all the html into one string which u then assign to the inner html

chandredewet commented on script.js:
```diff
@@ -1,12 +1,130 @@
+})  //displays episode depending on what option is picked
+ if(opt == ""){ 
+  displayNum.innerText = `Displaying ${allEpisodes.length}/${allEpisodes.length} episodes.`
+  return   rootElem.innerHTML = display();
```
Love ease of use in your code, displays understanding the concepts.

chandredewet reviewed (commented):
Hi Nishka, I have added some initial commentary but may add more in future.

Just a final question. Did you remove the setup Function because of the API level? This is normally so that the code will load first and users won't just see a blank browser.

Nishka-Kisten committed:
Changed to using the API

Nishka-Kisten committed:
Merge branch 'master' of https://github.com/Nishka-Kisten/tv-show-dom-project

Nishka-Kisten committed:
Displays all series

All series are able to be displayed, episode selector is not fixed yet. Still displays previous series that were clicked.

Nishka-Kisten committed:
Update script.js

Nishka-Kisten committed:
Edited CSS
